rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -332,6 +332,11 @@ namespace vk
 			enabled_features.shaderStorageImageWriteWithoutFormat = VK_TRUE;
 		}
 
+		if (g_cfg.video.precise_zpass_count)
```

Comment:
You then have to add a section below, checking if the hw reports the feature being supported.
If it is not supported, log an error and disable the feature. Follow this example: https://github.com/RPCS3/rpcs3/pull/10877/files#diff-9e6afc1b7246371e0560b474d9215274cd9b41757dce23fc35081633db74e35dR407 and add the check after it.

Reply:
I was thinking about it, I decided not to do it because it's only unsupported on phone GPUs such as Adreno, but okay.